### PR TITLE
Week10

### PR DESCRIPTION
**Pull request by Abdulaiman**

### CODE REVIEW

File: src/controllers/auth-controller.ts [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    if (!user) {
+      return next(customError("wrong email or password Try again", 404));
+    }
+    const correctPassword = await user?.correctPassword(
+      password,
+      user.password
+    );
+
+    if (!correctPassword) {
+      return next(customError("wrong password please try again", 404));
```
Comment (Leigh-K) [inline src/controllers/auth-controller.ts]: Thois error status should be a 403 forbidden

File: src/controllers/auth-controller.ts [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+      name,
+    }: { email: string; password: string; name: string } = req.body;
+
+    const user = await User.create({
+      email,
+      password,
+      name,
+    });
+    const token = jwt.sign({ id: user._id }, process.env.JWT_KEY, {
+      // expiresIn: process.env.JWT_EXPIRES_IN,
```
Comment (Leigh-K) [inline src/controllers/auth-controller.ts]: You should probably be using the EXPIRES for JWT, is this meant to be commented out

**Review by Leigh-K — APPROVED** [review]
Good work. Your user login and sign up controller return the password hashed. 
Its better to not send that back, just the pother user details PLUS token

File: src/controllers/auth-controller.ts (hunk 1 above)
Comment (Abdulaiman) [reply src/controllers/auth-controller.ts]: fixed thank you

File: src/controllers/auth-controller.ts (hunk 2 above)
Comment (Abdulaiman) [reply src/controllers/auth-controller.ts]: It was a mistake, i have fixed that too

**Pull request merged**